waggle-dance-core/pom.xml

```diff
@@ -259,8 +259,14 @@
     <dependency>
       <groupId>com.amazonaws.glue</groupId>
       <artifactId>aws-glue-datacatalog-hive2-client</artifactId>
-      <version>1.10.0-WD</version>
+      <version>1.10.0-WD-1</version>
     </dependency>
+    <dependency> <!-- Also used by redis -->
+      <groupId>com.esotericsoftware</groupId>
```

Comment:
This version is actually needed by redis but doesn't work for the glue datacatalog. For reads it's ok but doing an alter table throws an error: `Caused by: java.lang.ClassNotFoundException: com.esotericsoftware.kryo.pool.KryoFactory`
That's fine for now as we don't support and are still investigating Glue writes.